docs/quickstarts/chrome-extension.mdx

```diff
@@ -56,7 +56,7 @@ Add the SDK to your project:
 pnpm add @clerk/chrome-extension
 ```
 
-## Set your environment variables
+## Set your Clerk API keys
```

Comment:
This should probably keep the Frontend API URL as the user is copying one key and then one URL, versus keys like with most frameworks.